src/core/public/plugins/plugin.ts

```diff
@@ -71,7 +71,7 @@ export class PluginWrapper<
   private instance?: Plugin<TSetup, TStart, TPluginsSetup, TPluginsStart>;
 
   constructor(
-    readonly discoveredPlugin: DiscoveredPlugin,
+    public readonly discoveredPlugin: DiscoveredPlugin,
```

Comment:
This and other instances of adding `public` modifiers are necessary because of https://github.com/typescript-eslint/typescript-eslint/issues/857, which prevented property parameters created using the `readonly` keyword from being validated like normal.